translator/translate/otel/processor/awsappsignals/translator.go

```diff
@@ -56,16 +58,28 @@ func (t *translator) ID() component.ID {
 
 func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 	configKey := common.AppSignalsConfigKeys[t.dataType]
-	cfg := t.factory.CreateDefaultConfig().(*awsappsignals.Config)
+	cfg := t.factory.CreateDefaultConfig().(*appsignalsconfig.Config)
+
 	if common.IsAppSignalsKubernetes() {
-		cfg.Resolvers = []string{"eks"}
+		clusterNameFromMetricsConfig := common.ConfigKey(common.LogsKey, common.MetricsCollectedKey, common.KubernetesKey, "cluster_name")
```

Comment:
Is this the best thing to do here. We are now adding a hard dependency between app_signals and container_insights.
If we tell customers you have to specify the cluster_name in the `kubernetes` section, that is essentially them enabling container insights.

Reply:
Introduced `hosted_in` field into `app_signals`.